ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -70,9 +70,10 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
     // This is allows the react-router conversion to utilize the helpContentKey
```

Comment:
this only refers to 1 of the 3 subscriptions

Reply:
fixed - thanks!